internal/graph/graph_test.go

```diff
@@ -111,6 +111,17 @@ func TestGraphString(t *testing.T) {
 	require.Contains(t, g.String(), "*graph.Parent12 -> (object) obj: *graph.Parent12")
 }
 
+func TestCtorConflicts(t *testing.T) {
+	t.Parallel()
+	g := NewGraph()
+
+	err := g.InsertConstructor(threeObjects)
+	require.NoError(t, err)
+
+	err = g.InsertConstructor(oneObject)
+	require.Contains(t, err.Error(), "ctor: func() (*graph.Child1, error), object type: *graph.Child1: node already exist for the constructor")
+}
```

Comment:
What happens if you write a test to provide `oneObject`, then `oneObject` again?
The current functionality is that it will override, but I think with this code it will error out?